extra-multimedia/kodi/autobuild/prepare

```diff
@@ -0,0 +1,10 @@
+RESDIR="$SRCDIR/../repo-resources"
+
+abinfo "Copying l10n addons..."
+(
```

Comment:
This whole section could be boiled down to just one line, this is just ridiculous.

```
cp -rv "$RESDIR"/resource.language.* \
    "$SRCDIR"/addons
```